R/user_utils.R

```diff
@@ -177,3 +177,73 @@ yn_to_numeric <- function(arg) {
     TRUE ~ NA_real_
   )
 }
+
+#' Print `source` Objects
+#'
+#' @param x An `source` object
+#' @param ... If `indent = <numeric value>` is specified the output is indented
+#'   by the specified number of characters.
+#'
+#' @return No return value, called for side effects
+#'
+#' @author Stefan Bundfuss
+#'
+#' @keywords internal
+#' @family internal
```

Comment:
Should we make this known to the user?  Just thinking this might be nice to have in the Utilities sections